sdk/keyvault/keyvault-keys/samples/typescript/src/specificServiceAPIVersion.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Our Key Vault Keys client currently only supports the Azure Key Vault service versions `7.0` and `7.1-preview`,
+// which we specify as part of the options that can be passed through the constructor.
+// The latest supported API version is `7.1-preview`, which is available as the `LATEST_API_VERSION` export.
+
+const { DefaultAzureCredential } = require("@azure/identity");
+const { KeyClient, LATEST_API_VERSION } = require("@azure/keyvault-keys");
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+export async function main(): Promise<void> {
+  // DefaultAzureCredential expects the following three environment variables:
+  // - AZURE_TENANT_ID: The tenant ID in Azure Active Directory
+  // - AZURE_CLIENT_ID: The application (client) ID registered in the AAD tenant
+  // - AZURE_CLIENT_SECRET: The client secret for the registered application
+  const credential = new DefaultAzureCredential();
+
+  const vaultName = "<YOUR KEYVAULT NAME>";
+  const url = `https://${vaultName}.vault.azure.net`;
+  
+  // Users can specify an Azure Key Vault service API version.
+  // Here we're setting it to undefined so that we can show how we're using the LATEST_API_VERSION export
+  const apiVersion: string | undefined = undefined;
+  
+  const client = new KeyClient(url, credential, {
+    // Internally we default to the LATEST_API_VERSION export, just like how we're doing here:
+    apiVersion: apiVersion || LATEST_API_VERSION
```

Comment:
My feedback here is the similar to what I had in https://github.com/Azure/azure-sdk-for-js/pull/9647#discussion_r444010391

When the only difference between a newly added sample and an older one is a constructor overload, I don't see the value in adding the new sample.

Especially if we have already called out the use of the overload in the readme

Reply:
Sounds good! Thanks!

Reply:
Done here: https://github.com/Azure/azure-sdk-for-js/pull/9656/commits/4328be364b282d69858fa76a2f02b34a8290c5a2